.yarnrc

```diff
@@ -0,0 +1 @@
+ignore-engines true
```

Comment:
Why is this necessary?

Reply:
It is necessary due to a known issue with a dependency that bumped uses. (cson-parser)
https://github.com/groupon/cson-parser/issues/74

Reply:
Let's add a comment to explain this

Reply:
done